gorp.go

```diff
@@ -604,6 +642,115 @@ func (c *ColumnMap) SetMaxSize(size int) *ColumnMap {
 	return c
 }
 
+// IndexMap represents a mapping between a Go struct field and a single
+// index in a table.
+// Unique and MaxSize only inform the
+// CreateTables() function and are not used by Insert/Update/Delete/Get.
+type IndexMap struct {
+	// Index name in db table
+	IndexName string
+
+	// If true, " unique" is added to create index statements.
+	// Not used elsewhere
+	Unique bool
+
+	// Index type supported by Dialect
+	// Postgres:  B-tree, Hash, GiST and GIN.
+	// Mysql: Btree, Hash.
+	// Sqlite: nil.
+	IndexType string
```

Comment:
The difference between `Btree` and `B-tree` is just the naming right?
So could we find a way to have one identifier that works for both Postgres and MySQL, to make migration between the two seamless when using Btree?